src/Nelmio/Alice/Instances/Processor/Methods/ArrayValue.php

```diff
@@ -44,6 +44,7 @@ public function canProcess(ProcessableInterface $processable)
      */
     public function process(ProcessableInterface $processable, array $variables)
     {
+        /** @var array $values */
         $values = $processable->getValue();
```

Comment:
how come we have to precise the type here for `$values`, shouldn't this be specified in the doc of `::getValue()`?